src/components/action-bar/action-bar.tsx

```diff
@@ -348,7 +332,6 @@ export class ActionBar
       <ExpandToggle
         el={el}
         expanded={expanded}
-        // rename intl properties for 1.0
```

Comment:
can you create an issue to rename these functional component params? post 1.0 :)

Reply:
https://github.com/Esri/calcite-components/issues/6043